Makefile

```diff
@@ -1,55 +1,56 @@
+NEBULA_CMD_PATH = "./cmd/nebula"
 BUILD_NUMBER ?= dev+$(shell date -u '+%Y%m%d%H%M%S')
 GO111MODULE = on
 export GO111MODULE
 
 all:
-	make bin-linux
-	make bin-arm
-	make bin-arm6
-	make bin-arm64
-	make bin-darwin
-	make bin-windows
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-linux
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-arm
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-arm6
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-arm64
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-darwin
+	make NEBULA_CMD_PATH=${NEBULA_CMD_PATH} bin-windows
```

Comment:
hmm, why do we list these all as separate `make` commands instead of just making them dependencies of `all`, like:

    all: bin-linux bin-arm bin-arm6 bin-arm64 bin-darwin bin-windows

Seems like that would be simpler and then we don't have to worry about the env vars thing. Can you test that?